### PR TITLE
Removes Candidate.is(_any)_active(...).

### PR DESCRIPTION
**Pull request by boudewijn-tribler**
Instead of is_active(...) use: get_category(...) in (u"walk", u"stumble")

There is no replacement for is_any_active(...).  Instead the states of
a candidate in other communities no longer influenc each other.

### CODE REVIEW

**Comment by boudewijn-tribler** [comment]
This stuff was fixed in #76.  Waiting for that one to be accepted, no use in fixing it in two pulls.

File: community.py [hunk 1]
```diff
@@ -1128,7 +1128,7 @@ def acceptable_global_time(self):
         now = time()
 
         def acceptable_global_time_helper():
-            options = sorted(global_time for global_time in (candidate.get_global_time(self) for candidate in self._dispersy.candidates if candidate.in_community(self, now) and candidate.is_any_active(now)) if global_time > 0)
+            options = sorted(global_time for global_time in (candidate.get_global_time(self) for candidate in self._dispersy.candidates if candidate.get_category(self, now) in (u"walk", u"stumble")) if global_time > 0)
```
Comment (NielsZeilemaker) [inline community.py]: Maybe use dispersy_yield_verified_candidates instead.

File: dispersy.py [hunk 2]
```diff
@@ -4670,7 +4670,7 @@ def _stats_candidates(self):
                 if community.get_classification() == u"PreviewChannelCommunity":
                     continue
 
-                candidates = [candidate for candidate in self._candidates.itervalues() if candidate.in_community(community, now) and candidate.is_any_active(now)]
+                candidates = [candidate for candidate in self._candidates.itervalues() if candidate.get_category(community, now) in (u"walk", u"stumble")]
```
Comment (NielsZeilemaker) [inline dispersy.py]: Why are Introduced candidates not logged?

Comment (boudewijn-tribler) [inline dispersy.py]: _stats_candidates never logged intro candidates, I didn't want to change this behavior (it only logs the candidates that are usable, so 'walk' or 'stumble', i.e. previously called 'active')

_stats_detailed_candidates shows candidates in more detail, this one does include 'intro' candidates.

File: community.py (hunk 1 above)
Comment (boudewijn-tribler) [inline community.py]: Good spot.  Done.

File: dispersy.py (hunk 2 above)
Comment (NielsZeilemaker) [inline dispersy.py]: Might be nice to add a docstring mentioning this fact.

2013/7/5 Boudewijn notifications@github.com

Comment (boudewijn-tribler) [inline dispersy.py]: Nice :)

**Comment by boudewijn-tribler** [comment]
Thanks!

**Pull request merged**